app/Http/Review/ReviewController.php

```diff
@@ -56,9 +56,9 @@ public function create()
      */
     public function store(Request $request)
     {
-        $this->reviews->create($request->all());
+        if($request) {$this->reviews->create($request->all());
```

Comment:
Please add a space after the `if` keyword to follow the PSR-2 coding style guide.